multiversx_sdk_cli/dependencies/modules.py

```diff
@@ -253,18 +253,36 @@ def get_latest_release(self) -> str:
 
 class Rust(DependencyModule):
     def _do_install(self, tag: str) -> None:
-        rustup_path = self._get_rustup_path()
-        downloader.download("https://sh.rustup.rs", rustup_path)
-        utils.mark_executable(rustup_path)
+        installer_url = self._get_installer_url()
+        installer_path = self._get_installer_path()
+
+        downloader.download(installer_url, str(installer_path))
+        utils.mark_executable(str(installer_path))
+
         if tag:
             toolchain = tag
         else:
             toolchain = "nightly"
 
-        args = [rustup_path, "--verbose", "--default-toolchain", toolchain, "--profile",
+        args = [str(installer_path), "--verbose", "--default-toolchain", toolchain, "--profile",
                 "minimal", "--target", "wasm32-unknown-unknown", "--no-modify-path", "-y"]
         myprocess.run_process(args, env=self.get_env_for_install())
 
+    def _get_installer_url(self) -> str:
+        platform = workstation.get_platform()
+
+        if platform == "windows":
+            return "https://win.rustup.rs"
```

Comment:
Added, but doesn't work out of the box (downloads & certificates-related issues).